chromatic.config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "https://www.chromatic.com/config-file.schema.json",
+  "onlyChanged": true,
```

Comment:
Eg er usikker på om `onlyChanged` kanskje må settast `false` eller at vi må legge inn `externals: ["packages/css", "pachages/theme"]`, for å sørge for at visuelle testar blir kjørt på nytt ved endringer i css og tema. Tenker å teste dette etter vi har fått inn baseline snapshots, før den tid vil alle stories lage nye snapshots uansett